01-guessing_game/Guessinggame.java

```diff
@@ -0,0 +1,115 @@
+import java.util.Scanner;
+import java.lang.Math;
+
+class Guessinggame {
+
+
+  //Initial Number variables
+   public static void main(String[] args) {
+   String Name;
+   String oddorEven;
+   String decision;
+   int range;
+   int minimum;
+   int sugoiAnswer;
+   int attempts = 0;
+   int numberWrong = 0;
+   int maxnum;
```

Comment:
this could be written as a single line:
`int range, minimum, answer, attempts, numberWrong, maximum = 0;`